python_ta/transforms/type_inference_visitor.py

```diff
@@ -388,12 +388,17 @@ def visit_slice(self, node: astroid.Slice) -> None:
         if node.inf_type.getValue() == type(None):
             node.inf_type = TypeInfo(slice)
 
+    def visit_extslice(self, node: astroid.ExtSlice):
+        node.inf_type = TypeInfo(Tuple)
```

Comment:
This is better, but I actually want you to get the individual elements as well (so the example below would have an `ExtSlice` of type `Tuple[int, str]`.